### ISSUE TITLE
Recurring Error/Failure in VS Code

### ISSUE THREAD

**Issue by djtech2k-MS** [microsoft/vscode-azurearmtools#1062]
Does this occur consistently? Yes
Repro steps:
Running VS Code, after time passes, I get this error.  It is because the URL that is used by the powershell script to download the .Net core seems to be inaccessible.

Action: startArmLanguageServer
Error type: Error
Error Message: This extension requires .NET Core for full functionality, but we were unable to download and install a local copy for the extension. If this error persists, please see https://aka.ms/vscode-armtools-dotnet for troubleshooting tips. 
Details: dotnet.acquire returned undefined

Version: 0.13.0
OS: win32
OS Release: 10.0.19042
Product: Visual Studio Code
Product Version: 1.50.1
Language: en

<details>
<summary>Call Stack</summary>

```
Object.wrapError extension.bundle.js:166:60234
Object.<anonymous> extension.bundle.js:239:55780
fulfilled extension.bundle.js:239:54595
```

</details>

**Comment by djtech2k-MS** [comment]
After further review, I have found the following:

The dotnet-install.ps1 script is trying to download the dotnet binaries from this URL: https://dotnetcli.azureedge.net/dotnet/Runtime/3.1.9/dotnet-win-x64.3.1.9.zip, based on a query to some kind of Azure "Feed".  That path is not reachable so it continuously fails.  I modified the script to point it to a local path of that zip, and it does install.  The issue then is that when VS Code starts, it wipes out the binaries that were copied into place using the script.  So it removes the files from where it needs them.

From that point forward, VS Code continually fails to download the files.  To workaround this error, I used the static path under "dotnetAcquisitionExtension.existingDotnetPath" to set the path to my manually installed dotnet core install.

The root issue seems to be the install script is trying to download the zip file from a URL that is not available.
